arraycontext/impl/pytato/__init__.py

```diff
@@ -612,9 +612,11 @@ def call_loopy(self, program, **kwargs):
 
         return call_loopy(program, processed_kwargs, entrypoint)
 
-    def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
+    def compile(self, f: Callable[..., Any],
+                single_version_only: bool = False) -> Callable[..., Any]:
         from .compile import LazilyPyOpenCLCompilingFunctionCaller
-        return LazilyPyOpenCLCompilingFunctionCaller(self, f)
+        return LazilyPyOpenCLCompilingFunctionCaller(self,
```

Comment:
Maybe also handle the other sub-classes of BaseLazilyCompilingFunctionCaller?

Reply:
Added in 5ed25a5